docker-app/qfieldcloud/core/models.py

```diff
@@ -1189,8 +1189,9 @@ class Meta:
 
     has_restricted_projectfiles = models.BooleanField(
         default=False,
+        verbose_name=_("Restrict project files"),
         help_text=_(
-            "Restrict modifications of QGIS/QField projectfiles to managers and administrators."
+            "If enabled, modifications of QGIS/QField project files (.qgs, qgz) will be restricted to to managers and administrators."
```

Comment:
```suggestion
            "If enabled, modifications of QGIS/QField project files (.qgs, qgz) will be restricted to managers and administrators."
```

Double "to".

Also, this is a bit more than the label, see what it actually implies here: https://github.com/opengisch/QFieldCloud/blob/eb06ddda73f805b7502f216bcd521cd0ba137335/docker-app/qfieldcloud/filestorage/utils.py#L69-L108



```suggestion
            "If enabled, modifications of QGIS/QField funcitonal files (.qgs, .qmd) will be restricted to managers and administrators of the project."
```

Reply:
Sorry, no ~qmd~, qgd. My bad, updated the comment above.

Happy to adjust "functional files" to something better indeed. I just find QGIS project files migth be confusing.